src/components/new-request/search.vue

```diff
@@ -104,48 +104,50 @@
       <template v-else-if="isChangeName">
         <BusinessLookupFetch />
 
-        <!-- XPRO jurisdiction -->
-        <Jurisdiction
-          v-if="isChangeNameXpro"
-          md="4"
-        />
-        <CompanyType v-if="getEntityTypeCd && isNumberedEntityType" />
+        <template v-if="isNameChangeable">
```

Comment:
What does the page look like if you're trying to change the name of a business whose name is not changeable?

Reply:
It shows the Name Input field and the "Check this name" button

Reply:
Is the user stuck? Is it clear what they can/should do next? May need to follow up with Janis. Can you post a screenshot please?

Reply:
When a user searches for a business that does not exist:

![business-not-found](https://github.com/bcgov/namerequest/assets/122323255/9f0bf989-59fb-479f-98f7-ce6e95e64df4)

When a user searches for a business that cannot change name (this was while modifying the result to a legal type that cannot do that action). I ran the idea of adding a new error message on the business lookup field past Yui:

![business-cannot-change-name](https://github.com/bcgov/namerequest/assets/122323255/e416efea-ac09-469b-8d63-9507f1361a7f)

Reply:
Looks great!